docs/docs/installation/setup-ssh-tunneling.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Setup SSH Tunneling
+hide_title: true
+sidebar_position: 13
+version: 1
+---
+
+## SSH Tunneling
+
+1. Turn on feature flag
+    - Change (this value)[https://github.com/apache/superset/blob/master/superset/config.py#L489] to `True`
```

Comment:
This link is going to be broken very quickly. You can use a link to a specific revision:

https://github.com/apache/superset/blob/eb8386e3f0647df6d1bbde8b42073850796cc16f/superset/config.py#L489

But even better would be also including what feature flag should be changed:

```suggestion
    - Change (`SSH_TUNNELING`)[https://github.com/apache/superset/blob/eb8386e3f0647df6d1bbde8b42073850796cc16f/superset/config.py#L489] to `True`
```